force-app/main/default/lwc/geFormField/geFormFieldHelper.js

```diff
@@ -0,0 +1,17 @@
+import { isNotEmpty } from "c/utilCommon";
+
+const validityCheck = (formField) => {
+    let fieldIsValid = formField.checkFieldValidity();
+
+    if (formField.element !== null && formField.element.required) {
+        return isNotEmpty(formField.value)
+            && formField.value !== formField.CUSTOM_LABELS.commonLabelNone
```

Comment:
Not sure if it matters here, since this method is primarily about calling the checkFieldValidity method, but does this mean that this method returns false when someone changes a picklist back to the --None-- option?  Is that considered an invalid option for picklists?

Reply:
Yeah, '--None--' would be considered an invalid option for a required picklist. I believe we have another bug work item from a while back related to applying the custom validity/inline error for this.

Currently the user would receive a top-level error message saying something like, 'The following fields are required: Payment Method' for a required Payment Method picklist where the value is '--None--' but the field itself doesn't have an inline error.